pom.xml

```diff
@@ -29,12 +33,12 @@
             <version>2.2</version>
```

Comment:
Ok, you've added properties. Now, instead of using `<version>2.2</version>` you can use `<version>${junit.jupiter.version}</version>`